### ISSUE TITLE
Example fails to build: Message: alpha must be within ]0;pi[

### ISSUE THREAD

**Issue by stefanor** [diffpy/libobjcryst#19]
Attempting to update libobjcryst in Debian to `2022.1.4`, we found that the example (`examples/testlib.cpp`) won't build any more. Presumably since a55f20da4d4150011a610d1ff72a4783b303a466.

Knowing nothing about crystallography, I'm not sure how to fix the example.

**Comment by vincefn** [comment]
Ah, indeed the example was incorrect, the angles were in degrees and not radians - exactly what a55f20da4d4150011a610d1ff72a4783b303a466 is supposed to detect.

It should work now with 70ab0ba.

Note the real testing is done using the `pyobjcryst` test suite, which is much more extensive.

**Comment by stefanor** [comment]
> Note the real testing is done using the pyobjcryst test suite, which is much more extensive.

Yeah, this is just a smoke-test, really.

**Comment by stefanor** [comment]
Duh, I hadn't noticed those 90s. :)

**Comment by stefanor** [comment]
Do the 65s also refer to angles?

```
Create crystal with spacegroup: P 65 2 (0 0 1) / P 65 2 2 / P 65 2 (0 0 1) / 179-> P 65 2 (0 0 1)
CIF: Create Crystal=
Lookup of 'P 65 2 (0 0 1)' symbol failed, trying as Hall symbol.
LibCryst ++ exception thrown!!
  Message: alpha must be within ]0;pi[
```

**Comment by vincefn** [comment]
That one is much more embarrassing. The P 65 2 2 is the spacegroup in determining the symmetric positions starting from one (x,yz) (see https://www.cryst.ehu.es/cgi-bin/cryst/programs/nph-getgen?gnum=179), so nothing to do with the angles.

What happens here is that particular spacegroup does not need angles, because they are all 90 or 120° by symmetry. But the angle values are still passed, non-initialised as far as I can see, and the error is thrown up _before_ the program realises that the angles are actually constrained...

This will need to be fixed upstream in objcryst. Usually the CIF still list the angles even when they are fixed by symmetry, but this would be really bad when this is not the case.

**Comment by stefanor** [comment]
Thanks for that. I'll revert 70ab0baa06dbcccf5bb8bfc9187562db391add1e for now, in Debian.

**Comment by vincefn** [comment]
This is now fixed by 8a7643a8fc and 5861e1557a, the test runs OK under debian. Thanks for reporting this !
